tests/test_pds3file.py

```diff
@@ -2,6 +2,8 @@
 import pytest
 import os
 import numpy
+import gzip
```

Comment:
Travis CI fails for Python 2.6 (for gzip and bz2) and for Python 2.7 (bz2).
Working on resolving this issue with Python Versions 2.6 and 2.7.